Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
> By the way, semantically this has the same scope as dot access for attributes

It doesn't really though.  The best analogue is honestly the `->` vs `.` operators in C, except sort of inverted.  The `.` operator in C directly accesses a member, while the `->` operator dereferences a pointer and *then* accesses the member.  The `.` operator in Java accesses a member function and applies it, while the `::` operator accesses a member function and creates a pointer to it (without application).

In the C/C++ mode, both `.` and `->` are given scope `punctuation.accessor`.  In Java, `.` is (now) given `punctuation.accessor` while `::` is given `keyword.operator.method-reference`.  That's certainly a bit inconsistent.

By another perspective though, the `::` operator in Java is analogous to the `&` operator in C (creating a pointer), which is scoped as `keyword.operator.c`.